assets/blocks/course-outline/course-block/edit.js

```diff
@@ -93,15 +94,7 @@ const EditCourseOutlineBlock = ( {
 	attributes,
 	setAttributes,
 } ) => {
-	// Toggle legacy metaboxes.
-	useEffect( () => {
-		if ( attributes.isPreview ) return;
```

Comment:
I think that this check exists so the effect to not be called when a preview of a block is created. Previews are created when the user selects a block in the block selector and the popup window is displayed with a preview of a block. Additionally previews are created in the 'Styles' selector.

Does removing this check means that `sensei_toggleLegacyMetaboxes` will be called for previews?

Reply:
Hum, yeah! That can be a good optimization! Added it here: https://github.com/Automattic/sensei/pull/3818/commits/97f1f2da0f70b0a3020863d54525db7fdcc013b4